docs/data-management-on-carbondata.md

```diff
@@ -271,6 +271,7 @@ This tutorial is going to introduce all commands and data operations on CarbonDa
   * Before executing this command the old table schema and data should be copied into the new database location.
   * If the table is aggregate table, then all the aggregate tables should be copied to the new database location.
   * For old store, the time zone of the source and destination cluster should be same.
+  * If CarbonData table is stored in CarbonData HIVE meta store then refresh will not register the CarbonData table to HIVE meta store catalogue.
```

Comment:
Please change the point like below to convey the details of reason.

"If old cluster uses hive meta store refresh will not work as schema file is present in file system"

Reply:
@KanakaKumar if schema file is present in the file system then why refresh will not work.
Refresh will not work schema not present in the filesystem.

Reply:
Sorry I mean "If old cluster uses hive meta store, refresh will not work as schema file not exists in file system"

Reply:
Thanks @KanakaKumar for your reviews. I handled the comment. Please have a look